src/fieldFactory.js

```diff
@@ -26,8 +27,15 @@ export default (field, options) => {
     );
   }
 
-  if ('http://schema.org/email' === field.id) {
-    return <EmailField key={field.name} source={field.name} />;
+  switch (field.id) {
+    case 'http://schema.org/email':
+      return <EmailField key={field.name} source={field.name} />;
+
+    case 'http://schema.org/url':
+      return <UrlField key={field.name} source={field.name} />;
+
+    default:
```

Comment:
you don't really need the default then.

Reply:
If I don't add it ESLint emits a warning.